vectordb_bench/backend/clients/elastic_cloud/elastic_cloud.py

```diff
@@ -145,7 +147,16 @@ def search_embedding(
 
     def ready_to_search(self):
         """ready_to_search will be called between insertion and search in performance cases."""
-        pass
+        assert self.client is not None, "should self.init() first"
+        self.client.indices.refresh(index=self.indice)
+        force_merge_task_id = self.client.indices.forcemerge(index=self.indice, max_num_segments=1, wait_for_completion=False)['task']
```

Comment:
Does it mean to merge all vector data into a single segment?

Reply:
@liliu-z correct. All the benchmarks I can see here are "index everything" then "how do we do on searching after the index is built". This is the best practice for such a case.